File: frontend/src/app/models/entity/MovimientosFarmacia.ts
import { MovimientosFarmaciaDet } from './MovimientosFarmaciaDet';

export class MovimientosFarmacia{
     constructor (
        public movimfarid     ?: number,
        public hdgcodigo      ?: number,
        public esacodigo      ?: number,
        public cmecodigo      ?: number,
        public tipomov        ?: number,
        public movimfecha     ?: string,
        public usuario        ?: string,
        public soliid         ?: number,
        public bodegaorigen   ?: number,
        public bodegadestino  ?: number,
        public estid          ?: number,
        public proveedorid    ?: number,
        public orconumdoc     ?: number,
        public numeroguia     ?: number,
        public numeroreceta   ?: number,
        public fechadocumento ?: string,
        public cantidadmov    ?: number,
        public valortotal     ?: number,
        public cliid          ?: number,
        public fechagrabacion ?: string,
        public serviciocargoid ?: number,
        public guiatipodcto    ?: number,
        public foliourgencia   ?: number,
        public numeroboletacaja ?: number,
        public motivocargoid    ?: number,
        public pacambulatorio   ?: number,
        public tipoformuhcfar   ?: number,
        public cuentaid         ?: number,
        public clienterut ?: string,
        public clientepaterno?: string,
        public clientematerno?: string,
        public clientenombres?: string,
        public proveedorrut?: string,
        public ProveedorDesc    ?: string,
        public Servidor         ?: string,
        public movimudescr ?: string,
	    public bodegadescr?: string,
	    public bodegadestinodes?: string,
	    public comprobantecaja?: string,
	    public estadocomprobantecaja?: number,
	    public glosaestadocaja?: string,
	    public movimientosfarmaciadet ?: MovimientosFarmaciaDet[],
     

    ) {}
}

File: backend/Modulos/models/movimientosFarmacia.go
package models

// MovimientosFarmacia is...
type MovimientosFarmacia struct {
	MovimFarID               int                      `json:"movimfarid"`
	HdgCodigo                int                      `json:"hdgcodigo"`
	EsaCodigo                int                      `json:"esacodigo"`
	CmeCodigo                int                      `json:"cmecodigo"`
	MovTipo                  int                      `json:"tipomov"`
	MovimFecha               string                   `json:"movimfecha"`
	PiUsuario                string                   `json:"usuario"`
	SoliID                   int                      `json:"soliid"`
	BodegaOrigen             int                      `json:"bodegaorigen"`
	BodegaDestino            int                      `json:"bodegadestino"`
	EstID                    int                      `json:"estid"`
	ProveedorID              int                      `json:"proveedorid"`
	OrcoNumDoc               int                      `json:"orconumdoc"`
	NumeroGuia               int                      `json:"numeroguia"`
	NumeroReceta             int                      `json:"numeroreceta"`
	FechaDocumento           string                   `json:"fechadocumento"`
	CantidadMov              int                      `json:"cantidadmov"`
	ValorTotal               int                      `json:"valortotal"`
	CliID                    int                      `json:"cliid"`
	FechaGrabacion           string                   `json:"fechagrabacion"`
	ServicioCargoID          int                      `json:"serviciocargoid,string"`
	GuiaTipoDcto             int                      `json:"guiatipodcto"`
	FolioUrgencia            int                      `json:"foliourgencia"`
	NumBoleta                int                      `json:"numeroboletacaja"`
	MotivoCargoID            int                      `json:"motivocargoid"`
	PacAmbulatorio           string                   `json:"pacambulatorio"`
	TipoFormuHCFAR           int                      `json:"tipoformuhcfar"`
	CuentaID                 int                      `json:"cuentaid"`
	ClienteRut               string                   `json:"clienterut"`
	ClientePaterno           string                   `json:"clientepaterno"`
	ClienteMaterno           string                   `json:"clientematerno"`
	ClienteNombres           string                   `json:"clientenombres"`
	ProveedorRUT             string                   `json:"proveedorrut"`
	ProveedorDesc            string                   `json:"proveedordesc"`
	MovimDescr               string                   `json:"movimudescr"`
	BodegaDescr              string                   `json:"bodegadescr"`
	BodegaDestinoDes         string                   `json:"bodegadestinodes"`
	MovComprobanteCaja       string                   `json:"comprobantecaja"`
	MovEstadoComprobanteCaja int                      `json:"estadocomprobantecaja"`
	GlosaEstadoCaja          string                   `json:"glosaestadocaja"`
	Servidor                 string                   `json:"servidor"`
	MovFarIDDespachoDevArt   int                      `json:"movfaridedspachodevart"`
	Detalle                  []MovimientosFarmaciaDet `json:"movimientosfarmaciadet"`
}
